spot_rl_experiments/spot_rl/envs/gaze_env.py

```diff
@@ -46,7 +42,9 @@ def parse_arguments(args=sys.argv[1:]):
     return args
 
 
-def construct_config_for_gaze(file_path=None, opts=[]):
+def construct_config_for_gaze(
+    file_path=None, opts=[], dont_pick_up=False, max_episode_steps=None
```

Comment:
why are these extra parameters instead of in opts?

Reply:
Seemed to me cleanest that way. As discussed offline, this is okay.